### PR TITLE
CardanoCSL: fetch node0 public IP

### PR DESCRIPTION
**Pull request by domenkozar**
cc @georgeee

### CODE REVIEW

File: CardanoLib.hs [hunk 1]
```diff
@@ -69,7 +79,50 @@ scpFromNode args node from to = do
 
 sshForEach args cmd = nixops <> "ssh-for-each" <> args <> " -- " <> cmd
 
+-- Functions for extracting information out of nixops info command
+
 -- | Get all node IPs in EC2 cluster
+getNodes :: NixOpsArgs -> IO [Text]
```
Comment (georgeee) [inline CardanoLib.hs]: Please filter out 'obsolete' nodes

Comment (domenkozar) [inline CardanoLib.hs]: When is a node obsolete?

Comment (georgeee) [inline CardanoLib.hs]: When it's deployed and then commented out from nic configuration without destroying. Sometimes it's useful. This nodes are marked in `nixops info` output. Please try commenting out some nodes and check `info` output

Comment (domenkozar) [inline CardanoLib.hs]: Thanks! I'll check.

**Pull request merged**